ICLGeom/src/ICLGeom/Scene.cpp

```diff
@@ -137,7 +133,7 @@ namespace icl{
             which can of course lead to any issues!
             */
         try{
-          PlaneEquation p(T*Vec(0,0,S,1),T*Vec(0,0,1,1)-cam.getPosition());
```

Comment:
Both code snippets do exactly the same. Former required subtracting cam position, because it was added by homogenous point vector. Using a direction vector (vec[3]=0), position isn't added in first place.